### DISCUSSION TITLE
Rasberry Pi set up questions

### DISCUSSION THREAD

**Discussion by chiachain** [Chia-Network/chia-blockchain#1678] [Q&A]
Hello all. So, I want to set up my new Rasberry Pi 400 (fresh out of the box), and could use some help (as in....I have no idea what I am doing).

It needs 64 bit, but system is 32 bit.  Directions say # If you are not using Raspbian 64 (it pre-specifies it) add this
export PIP_EXTRA_INDEX_URL=https://www.piwheels.org/simple/

I copied and pasted this in the terminal but nothing happens.  Or....Do I have to change the 32 bit to 64 bit?  Seems like a massive undertaking for a beginner.  Do I also have to do this swap space, run on Rasberry Pi imager...Ubantu servers......etc.???  

For a beginner, is there a much more detailed explanation anywhere?  A youtube vid would be GREAT!!!!!

**Comment by hoffmang9** [comment]
You need to use the Raspberry Pi installer to choose one of the 64bit options. https://github.com/Chia-Network/chia-blockchain/wiki/Raspberry-Pi
